docs/UserGuide.md

```diff
@@ -224,28 +261,39 @@ Example of usage:
 * `timetable -add CS2101 TUESDAY 0800 1000 LECTURE 0`
 * `timetable -add CS2113T MONDAY 1300 1500 LAB 1`
 
-#### 4.11.3 Delete a lesson: `timetable -del`
+#### 3.11.3 Delete a lesson: `timetable -del`
 
 Deletes a lesson from the timetable.\
 Format: `timetable -del <day> <lesson index>`  
 `<day>` : Must be one of the following : `MONDAY`, `TUESDAY`, `WEDNESDAY`, `THURSDAY`, `FRIDAY`, `SATURDAY`, `SUNDAY`
 `<lesson index>` : Index of the lesson to be deleted. See timetable and the indexes with `timetable -day` or `timetable -week`
 
 Example of usage: 
-* `timetable -del CS2101 1`
+* `timetable -del MONDAY 1`
 
-### 4.12 Exiting the program: `bye`
+### 3.12 Exiting the program: `bye`
 
 Exits ra.VI.\
 Format: `bye`
 
-## FAQ
+## 4 FAQ
+
+**Q**: Can I use ra.VI on operating system other than Windows?\
+**A**: Yes. ra.VI is compatible with Windows, macOS and Linux.
+
+**Q**: Can I use ra.VI throughout my time in NUS?\
+**A**: Yes! In fact, we strongly encourage using ra.VI for your stint at NUS. ra.VI is robust enough to handle the trials of time.
```

Comment:
"...time at NUS" will probably be better rather than "stint"

Reply:
Ok, amended